Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -571,8 +571,20 @@ def audit_specs
         end
       end
 
+      head_msg = "Formulae should not have a HEAD spec"
+
       if @new_formula && formula.head
-        new_formula_problem "Formulae should not have a HEAD spec"
+        return if formula.versioned_formula?
```

Comment:
This would exit the entire function skipping remaining checks below but this line isn't needed at all if we're going to skip all @new_formula audits for versioned formulae.

Reply:
> if we're going to skip all @new_formula audits for versioned formulae.

How?

Reply:
Something like
```
      @new_formula = options[:new_formula] && !formula.versioned_formula?
```
I suppose?